doc/nrf/releases_and_maturity/releases/release-notes-2.9.0.rst

```diff
@@ -0,0 +1,661 @@
+    * The ``startup_cmd`` command.
+      This command is used to store shell commands to be run sequentially after bootup.
+    * Band 4 support for nRF9151 with modem firmware v1.0.2.
+
+  * Updated:
+
+    * The ``dect rssi_scan`` command with busy/possible/free subslot count-based RSSI scan.
+    * The ``dect rx`` command to provide the possibility to iterate all channels and to enable RX filter.
+
+Edge Impulse samples
+--------------------
+
+* Added support for the :ref:`zephyr:nrf54h20dk_nrf54h20` board target in the following samples:
+
+  * :ref:`ei_data_forwarder_sample`
+  * :ref:`ei_wrapper_sample`
+
+Matter samples
+--------------
+
+* Updated:
+
+  * All Matter samples that support low-power mode to enable the :ref:`lib_ram_pwrdn` feature.
+    It is enabled by default for the release configuration of the following samples:
+
+    * :ref:`matter_lock_sample`
+    * :ref:`matter_light_switch_sample`
+    * :ref:`matter_smoke_co_alarm_sample`
+    * :ref:`matter_window_covering_sample`
+
+  * All Matter samples to enable the ZMS file subsystem in all devices that contain MRAM, such as the nRF54H Series devices.
+
+* Disabled pausing Matter watchdog while CPU is in idle state in all Matter samples.
+  To enable it, set the :ref:`CONFIG_NCS_SAMPLE_MATTER_WATCHDOG_PAUSE_IN_SLEEP<CONFIG_NCS_SAMPLE_MATTER_WATCHDOG_PAUSE_IN_SLEEP>` Kconfig option to ``y``.
+
+* :ref:`matter_template_sample` sample:
+
+  * Added support for the ``nrf54l15dk/nrf54l10/cpuapp`` board target.
+  * Updated the internal configuration for the :ref:`zephyr:nrf54l15dk_nrf54l15` target to use the DFU image compression and provide more memory space for the application.
+
+* :ref:`matter_smoke_co_alarm_sample` sample:
+
+  * Added:
+
+    * Support for ICD dynamic SIT LIT switching (DSLS).
+    * Support for the ``nrf54l15dk/nrf54l10/cpuapp`` board target.
+
+Peripheral samples
+------------------
+
+* Added support for the ``nrf54l15dk/nrf54l05/cpuapp`` and ``nrf54l15dk/nrf54l10/cpuapp`` board targets in the :ref:`radio_test` sample.
+
+Protocol serialization samples
+------------------------------
+
+* Updated GPIO pins on the nRF54L15 DK used for communication between the client and server over UART.
+  One of the previously selected pins was also used to drive an LED, which may have disrupted the UART communication.
+
+Thread samples
+--------------
+
+* Removed support for the ``nrf5340dk/nrf5340/cpuapp/ns`` board target for all samples.
+
+* :ref:`ot_cli_sample` sample:
+
+  * Added support for the ``nrf54l15dk/nrf54l10/cpuapp`` board target.
+
+* :ref:`ot_coprocessor_sample` sample:
+
+  * Added support for the ``nrf54l15dk/nrf54l10/cpuapp`` and ``nrf54l15dk/nrf54l05/cpuapp`` board targets.
+
+Other samples
+-------------
+
+* :ref:`coremark_sample` sample:
+
+  * Updated:
+
+    * Configuration for the :ref:`zephyr:nrf54h20dk_nrf54h20` board to support multi-domain logging using the ARM Coresight STM.
```

Comment:
Hi @peknis, please move this entry back to the changelog, it is relevant for 2.9-nRF54H20.